packages/core/package.json

```diff
@@ -76,7 +76,7 @@
     "mocha-test": "mocha ./mocha-test --recursive",
     "load-addresses": "yarn run apply-registry",
     "clean": "rm -rf build",
-    "build": "yarn truffle compile && yarn load-addresses && yarn generate-contract-types && yarn compile-ts && yarn buildLatestHardhatVersionHashes",
+    "build": "yarn truffle compile && yarn load-addresses && yarn generate-contract-types && yarn compile-ts && yarn buildLatestHardhatVersionHashes && yarn hardhat compile --network optimism",
```

Comment:
Added the `compile --network optimism` task to `build`. We don't test this (as this would require running a local optimism node, which is not easy to do yet since the only way is to run 6 docker containers), but worth just compiling for now.